### DISCUSSION TITLE
How to enable amp training?

### DISCUSSION THREAD

**Discussion by jsrdcht** [yoshitomo-matsubara/torchdistill#385] [Q&A] [answered]
I found nothing in the library

**Comment by yoshitomo-matsubara** [comment, marked as answer]
Hi @jsrdcht 

See https://github.com/yoshitomo-matsubara/torchdistill/discussions/247

**Reply by jsrdcht** [reply]
Thanks. This is my first time using huggingface, and this library _accelerate_ seems very interesting. Although learning a new codebase can be painful.
